app/controllers/hq/users_controller.rb

```diff
@@ -6,4 +6,24 @@ def index
     @users = User.all
   end
 
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update_attributes(user_params)
+      flash[:success] = 'User successfully saved'
+      redirect_to hq_user_url(@user)
+    else
+      render 'edit'
+    end
+  end
+
+private
+
+  def user_params
+    params[:user].permit(:user_name, :email)
```

Comment:
No need to do anything. It would be impossible to trigger this exception if the app is built and tested correctly, and if the end user operates via the UI. If either of those isn't the case, it's appropriate for the app to fail loudly.